nixos/tests/seafile.nix

```
@@ -117,5 +123,8 @@ import ./make-test-python.nix ({ pkgs, ... }:
           client2.succeed("ls -la test01 >&2")
 
           client2.succeed('[ `cat test01/first_file` = "bla" ]')
+
+      with subtest("seafdav query"):
+          client1.succeed("curl --anyauth --user 'admin@myrdd.info:seafile_password' http://server/seafdav | grep 'Authenticated user' >&2")
```

This should match declared user:
```suggestion
          client1.succeed("curl --anyauth --user 'admin@example.com:seafile_password' http://server/seafdav | grep 'Authenticated user' >&2")
```